### DISCUSSION TITLE
Welcome to Papercups Discussions!

### DISCUSSION THREAD

**Discussion by reichert621** [papercups-io/papercups#458] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aguilaair** [comment]
Hey👋!

I'm Eduardo, I'm a spanish devloper and I love to play with my dog. I'm currently developing the flutter widget for papercups 😄.

**Reply by reichert621** [reply]
Loving your work on the Flutter library @aguilaair! 🎉

**Comment by reuben581** [comment]
Hello my name is Reuben ukoli I want to be a very good developer

**Reply by reichert621** [reply]
Welcome!

**Comment by jaimeiniesta** [comment]
Hello! I'm Jaime, from Spain, and I'm interested in using Papercups as a cookie-free alternative to what we were using previously, Tawk. Yesterday I wrote [this post about being privacy-conscious](https://rocketvalidator.com/blog/privacy-conscious) and looking for alternatives to popular 3rd party services that use tracking cookies.

**Reply by reichert621** [reply]
Thanks for the great writeup! Let us know if there's anything we can do to make Papercups a better experience for you :)

**Reply by jaimeiniesta** [reply]
Thanks! What I'm missing most right now, coming from Tawk.to, is a better Live Sessions page. I'd like to see the country that each current visitor comes from, directly on the table or maybe a World map like in web analytics.

Also, I'm missing a log of the URLs that each visitor has passed through, not just the last one.
